pyquil/kraus.py

```diff
@@ -56,3 +58,211 @@ def _create_kraus_pragmas(name, qubit_indices, kraus_ops):
                       "({})".format(" ".join(map(format_parameter, np.ravel(k)))))
                for k in kraus_ops]
     return pragmas
+
+
+def append_kraus_to_gate(kraus_ops, gate_matrix):
```

Comment:
Are all of these functions public?

Reply:
Open to suggestions. I'd imagine most of the kraus low-level functions might be generally useful. Maybe `noisy_instruction` should be private

Reply:
Agree the noisy_instruction should be private. In general anything that operates on low-level PyQuil program representations shouldn't be too public